Source/WebGPU/WebGPU/RenderPipeline.mm

```diff
@@ -545,7 +545,7 @@ static void populateStencilOperation(MTLStencilDescriptor *mtlStencil, const WGP
     auto bindGroupLayout = m_device->createBindGroupLayout(bindGroupLayoutDescriptor);
     m_cachedBindGroupLayouts.add(groupIndex + 1, bindGroupLayout);
 
-    return bindGroupLayout.ptr();
+    return WebGPU::releaseToAPI(WTFMove(bindGroupLayout));
```

Comment:
I think the releaseToAPI calls should be in the wgpuFoo() functions themselves

Reply:
In general I agree, but this `getBindGroupLayout` sometimes creates and sometimes returns without creating (line 513). So we can't release it in the wgpu if it was not created, right?

Reply:
Oh, in that case the newly created one should be internally retained (so a second call gets the same object).

Reply:
Doesn't that happen on line 546 or did I do that wrong?